tests/downloadNightlyZip.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+if [[ -z "${PS_TARGET_VERSION}" ]]; then
+    echo "Variable PS_TARGET_VERSION must be defined"
+    exit 1
+fi
+
+if [[ -z "${PS_TARGET_BRANCH}" ]]; then
+    echo "Variable PS_TARGET_BRANCH must be defined"
+    exit 1
+fi
+
+
+YESTERDAY="$(date -d yesterday '+%Y-%m-%d')"
```

Comment:
Are you sure a build is done in week end ?

Reply:
This is one of the current limitations of this PR. In theory a build is computed every night, weekend included. In practice sometimes the build process fails.

A good solution will be
> create a API endpoint on nightly.prestashop.com that would return the latest available ZIP files and we can use them instead of "guessing" the download URL ; it could also provide the data needed by UpdateConfig

But this is another repo, another PR, another project 😅

Reply:
But have you checked, we use this API every day from Presthubot :

https://api-nightly.prestashop.com/reports

Reply:
Ooooh ❤️  that is a very good starting point indeed.

I see however this returns very old reports like 1.7.6.x reports so I will need nonetheless to create a 2nd endpoint with just the latest results. But this is great: I should be able to reuse the logic behind this to build it.

Reply:
@Progi1984 Parsing the big JSON returned by https://api-nightly.prestashop.com/reports will require a small parser (I will not do this in bash 😆 ). I would like to avoid this PR getting any bigger and implement this in another PR as an improvement.

Reply:
@matks Use `jq` (https://stedolan.github.io/jq/) and your life will be more easier ;)

Reply:
(and it's available on Travis : https://travis-ci.com/github/PrestaShop/autoupgrade/jobs/450892848#L52)

Reply:
Done :

```bash
$ curl -sS https://api-nightly.prestashop.com/reports | jq -r 'first(.[] | select(.version == "1.7.6.x")) | .download'
null
$ curl -sS https://api-nightly.prestashop.com/reports | jq -r 'first(.[] | select(.version == "1.7.7.x")) | .download'
https://storage.googleapis.com/prestashop-core-nightly/2020-12-01-1.7.7.x-prestashop_1.7.7.0.zip
$ curl -sS https://api-nightly.prestashop.com/reports | jq -r 'first(.[] | select(.version == "develop")) | .download'
https://storage.googleapis.com/prestashop-core-nightly/2020-12-01-develop-prestashop_1.7.8.0.zip

```